rust/osm2lanes/src/transform/tags_to_lanes/lane.rs

```diff
@@ -0,0 +1,266 @@
+use super::{Infer, Oneway};
+use crate::locale::Locale;
+use crate::road::Direction;
+use crate::tag::{TagKey, Tags};
+use crate::transform::tags_to_lanes::modes::{BuswayScheme, LanesBusScheme};
+use crate::transform::{RoadWarnings, TagsToLanesMsg};
+
+const LANES: TagKey = TagKey::from("lanes");
+#[derive(Debug)]
+pub struct LanesScheme {
```

Comment:
the name of the mod and this struct are unclear for me

in my PR I named this `Counts`, as it isn't really defining a scheme?

Reply:
I didn't know what to name the mod. The struct is named `LanesScheme` because it represents the `lanes:*` tags (which together make up what I was calling "scheme").